apps/perf-test/src/scenarios/FocusTrapZone.tsx

```diff
@@ -0,0 +1,19 @@
+import * as React from 'react';
```

Comment:
Do we really need a perf test for this? Each perf test does have a cost, so I'm not sure we necessarily need them for all components by default.